outputs.tf

```diff
@@ -1,14 +1,31 @@
-output "ecs_cluster_id" {
-  description = "ID of the ECS Cluster"
-  value       = concat(aws_ecs_cluster.this.*.id, [""])[0]
+################################################################################
+# Cluster
+################################################################################
+
+output "cluster_arn" {
+  description = "ARN that identifies the cluster"
+  value       = try(aws_ecs_cluster.this[0].arn, null)
 }
 
-output "ecs_cluster_arn" {
-  description = "ARN of the ECS Cluster"
-  value       = concat(aws_ecs_cluster.this.*.arn, [""])[0]
+output "cluster_id" {
+  description = "ID that identifies the cluster"
+  value       = try(aws_ecs_cluster.this[0].id, null)
 }
 
-output "ecs_cluster_name" {
-  description = "The name of the ECS cluster"
-  value       = var.name
+################################################################################
+# Cluster Capacity Providers
+################################################################################
+
+output "cluster_capacity_providers" {
+  description = "Map of cluster capacity providers attributes"
+  value       = aws_ecs_cluster_capacity_providers.this
```

Comment:
The normal output in the docs is just the `id` which is the cluster ID and not useful. However, this could be useful because it shows the aggregate view of the capacity providers and their default capacity provider strategy settings - both Fargate and EC2 AutoScaling

Reply:
Maybe prepared data would be a bit more useful here? For example, instead of returning the whole resources, group them by capacity provider name? I don't know which property of these resources can be used for real. 

Here is an example from ALB module - https://github.com/terraform-aws-modules/terraform-aws-alb/blob/master/outputs.tf#L63-L65

Reply:
Updated to map ID to output. Its nothing fancy, its mostly a summary view of the capacity providers and their strategies for the cluster. Heres the example output now:

```hcl
cluster_capacity_providers = {
  "ecs-ex-complete" = {
    "capacity_providers" = toset([
      "one",
      "two",
    ])
    "cluster_name" = "ecs-ex-complete"
    "default_capacity_provider_strategy" = toset([
      {
        "base" = 0
        "capacity_provider" = "two"
        "weight" = 40
      },
      {
        "base" = 20
        "capacity_provider" = "one"
        "weight" = 60
      },
    ])
    "id" = "ecs-ex-complete"
  }
}
```